src/templates/next/layouts/Skeleton/Skeleton.tsx

```diff
@@ -35,13 +39,11 @@ export const Skeleton = ({
           type: "navbar",
           logoUrl: site.logoUrl,
           logoAlt: site.siteName,
-          search: {
-            type: "localSearch",
-            searchUrl: "/search",
-          },
+          search: site.search,
```

Comment:
don't have to be in this PR but I guess we would eventually want to move these Skeleton components to `/shared` and stop using `renderComponent` for them